static/src/javascripts/projects/common/modules/commercial/hosted-video.js

```diff
@@ -85,99 +85,103 @@ define([
     }
 
     function init() {
-        require(['bootstraps/enhanced/media/main'], function () {
```

Comment:
Oh right, ok who goes first? Just make sure you replace the `return` statement with a call to `resolve()` instead